cf_units/__init__.py

```diff
@@ -62,8 +62,6 @@
            'UT_ASCII',
            'FLOAT32',
            'FLOAT64',
-           'julian_day2date',
-           'date2julian_day',
```

Comment:
There are also entries in [utilities.rst](https://github.com/SciTools/cf-units/blob/master/doc/source/utilities.rst) that need removing.

Reply:
Done